geomesa-spark/geomesa-spark-accumulo/src/test/scala/org/locationtech/geomesa/spark/accumulo/SparkSQLDataTest.scala

```diff
@@ -0,0 +1,140 @@
+
+import com.vividsolutions.jts.geom.{Coordinate, Point}
+import org.apache.accumulo.minicluster.MiniAccumuloCluster
+import org.apache.spark.sql.{DataFrame, SQLContext, SparkSession}
+import org.geotools.data.DataStoreFinder
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.AccumuloProperties.AccumuloQueryProperties
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+import org.locationtech.geomesa.index.conf.QueryProperties
+import org.locationtech.geomesa.utils.text.WKTUtils
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class SparkSQLDataTest extends Specification {
+  val createPoint = JTSFactoryFinder.getGeometryFactory.createPoint(_: Coordinate)
+
+  "sql data tests" should {
+    sequential
+
+    System.setProperty(QueryProperties.SCAN_RANGES_TARGET.property, "1")
+    System.setProperty(AccumuloQueryProperties.SCAN_BATCH_RANGES.property, s"${Int.MaxValue}")
+
+    var mac: MiniAccumuloCluster = null
+    var dsParams: JMap[String, String] = null
+    var ds: AccumuloDataStore = null
+    var spark: SparkSession = null
+    var sc: SQLContext = null
+
+    var df: DataFrame = null
+    var gndf: DataFrame = null
+    var sdf: DataFrame = null
+
+    // before
+    step {
+      mac = SparkSQLTestUtils.setupMiniAccumulo()
+      dsParams = SparkSQLTestUtils.createDataStoreParams(mac)
+      ds = DataStoreFinder.getDataStore(dsParams).asInstanceOf[AccumuloDataStore]
+
+      spark = SparkSession.builder().master("local[*]").getOrCreate()
+      sc = spark.sqlContext
+      sc.setConf("spark.sql.crossJoin.enabled", "true")
+    }
+
+    "ingest chicago" >> {
+      SparkSQLTestUtils.ingestChicago(ds)
+
+      df = spark.read
+        .format("geomesa")
+        .options(dsParams)
+        .option("geomesa.feature", "chicago")
+        .load()
+      df.printSchema()
+      df.createOrReplaceTempView("chicago")
+
+      df.collect.length mustEqual 3
+    }
+
+    "ingest geonames" >> {
+      SparkSQLTestUtils.ingestGeoNames(dsParams)
+
+      gndf = spark.read
+        .format("geomesa")
+        .options(dsParams)
+        .option("geomesa.feature", "geonames")
+        .load()
+      gndf.printSchema()
+      gndf.createOrReplaceTempView("geonames")
+
+      gndf.collect.length mustEqual 2550
+    }
+
+    "ingest states" >> {
+      import org.apache.spark.sql.functions.broadcast
+
+      SparkSQLTestUtils.ingestStates(ds)
+
+      sdf = spark.read
+        .format("geomesa")
+        .options(dsParams)
+        .option("geomesa.feature", "states")
+        .load()
+      sdf.printSchema()
+      sdf.createOrReplaceTempView("states")
+
+      broadcast(sdf).createOrReplaceTempView("broadcastStates")
+
+      sdf.collect.length mustEqual 56
+    }
+
+    "basic sql 1" >> {
+      val r = sc.sql("select * from chicago where case_number = 1")
+      r.show()
```

Comment:
does this print to std out? should probably hide this in a debug statement or something if possible, or remove if not